src/cuttlefish_escript.erl

```diff
@@ -81,7 +81,7 @@ main(Args) ->
     application:set_env(lager, handlers, [{lager_stderr_backend, LogLevel}]),
     application:set_env(lager, crash_log, false),
     lager:start(),
-    lager:debug("Cuttlefish set to debug level logging"),
+    lager:log(debug,self(),"Cuttlefish set to debug level logging"),
```

Comment:
parse transform'd Erlang code won't work in elixir from 1.6 onwards - as there is minimal use of the lager parse transform it seemed reasonable to switch to direct lager calls